src/js/utils/helpers.js

```diff
@@ -0,0 +1,15 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+export const isMobile = (() => {
+
+  let hasTouchScreen = false;
+
+  try {
+    hasTouchScreen = typeof window.orientation !== 'undefined';
```

Comment:
this would fail to recognize touch screen with server side rendering. 

The app will not fail because you added a catch here. But hasTouchScreen would base false, when one would expect that to be true.